src/test/java/teammates/test/cases/browsertests/FeedbackRubricQuestionUiTest.java

```diff
@@ -394,25 +387,79 @@ public void testEditQuestionAction() throws Exception {
-        assertTrue(feedbackEditPage.moveRubricColRight(-1, 1));
-        assertTrue(feedbackEditPage.moveRubricColRight(-1, 2));
-        feedbackEditPage.clickAddQuestionButton();
+        moveSecondRubricColumnToLast(-1, colIndexes, col5, col0, col1, col2, col3);
 
+        feedbackEditPage.clickAddQuestionButton();
         feedbackEditPage.verifyStatus(Const.StatusMessages.FEEDBACK_QUESTION_ADDED);
         feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackRubricQuestionMoveColumnNewQuestionSuccess.html");
         feedbackEditPage.clickDeleteQuestionLink(2);
         feedbackEditPage.waitForConfirmationModalAndClickOk();
     }
 
+    /**
+     * Moves the last rubric column to the first position. Accomplishes this by
+     * clicking move column left button for last, second-last upto second column.
+     * Verifies the rubric question after each move.
+     * @param qnNumber question number.
+     * @param colIndexes An array containing column indexes in the order displayed in the UI.
+     * @param columns Varargs parameter, where each parameter is {@code String[]} which denotes values
+     *         of a rubric column. Column values must be given in the order displayed in the UI.
+     */
+    private void moveLastRubricColumnToFirst(int qnNumber, int[] colIndexes, String[]... columns) {
+        Assumption.assertEquals(colIndexes.length, columns.length);
+
+        for (int i = colIndexes.length - 1; i >= 1; i--) {
+            assertTrue(feedbackEditPage.moveRubricColLeft(qnNumber, colIndexes[i]));
+
+            // swap i, and i - 1 column
+            String[] temp = columns[i];
+            columns[i] = columns[i - 1];
+            columns[i - 1] = temp;
+
+            feedbackEditPage.verifyRubricQuestion(qnNumber, colIndexes, columns);
+        }
+    }
+
+    /**
+     * Moves the second rubric column to the last position. Accomplishes this by
+     * clicking move column right button for second, third so on upto second-last
+     * column. Verifies the rubric question after each move.
+     * @param qnNumber question number.
+     * @param colIndexes An array containing column indexes in the order displayed in the UI.
+     * @param columns Varargs parameter, where each parameter is {@code String[]} which denotes values
+     *         of a rubric column. Column values must be given in the order displayed in the UI.
+     */
+    private void moveSecondRubricColumnToLast(int qnNumber, int[] colIndexes, String[]... columns) {
+        Assumption.assertEquals(colIndexes.length, columns.length);
+
+        for (int i = 1; i < colIndexes.length - 1; i++) {
+            assertTrue(feedbackEditPage.moveRubricColRight(qnNumber, colIndexes[i]));
+
+            // swap i, and i + 1 column
+            String[] temp = columns[i];
+            columns[i] = columns[i + 1];
+            columns[i + 1] = temp;
+
+            feedbackEditPage.verifyRubricQuestion(qnNumber, colIndexes, columns);
+        }
+    }
```

Comment:
These two methods cannot be unified? i.e. specify as parameters which column to move to where.